Code/Editor/AzAssetBrowser/AzAssetBrowserWindow.cpp

```diff
@@ -115,6 +115,14 @@ AzAssetBrowserWindow::AzAssetBrowserWindow(QWidget* parent)
         m_tableModel->setDynamicSortFilter(true);
         m_ui->m_assetBrowserTableViewWidget->setModel(m_tableModel.data());
 
+        m_createMenu = new QMenu("Create Menu", this);
```

Comment:
nit: "Create Menu" may be easy to misunderstand, maybe it's better to label this "Create File Menu"?

Reply:
@AMZN-daimini I'll change it to "Create New Asset Menu" to make it even easier😀